pyenvisalink/mock_server_dsc.py

```diff
@@ -47,51 +46,49 @@ async def process_command(self, line) -> bool:
             return False
 
         success = False
-        if cmd == "000": # Poll
+        if cmd == "000":  # Poll
             success = await self.poll()
-        elif cmd == "001": # Status Report
+        elif cmd == "001":  # Status Report
             success = await self.status_report()
-        elif cmd == "008": # Dump Zone Timers
+        elif cmd == "008":  # Dump Zone Timers
             success = await self.dump_zone_timers()
-        elif cmd == "005": # Network Login
+        elif cmd == "005":  # Network Login
             success = await self.login(data)
-        elif cmd == "010": # Set Time & Date
+        elif cmd == "010":  # Set Time & Date
             success = await self.set_time_and_date()
-        elif cmd == "020": # Command Output Control
+        elif cmd == "020":  # Command Output Control
             success = await self.command_output_control()
-        elif cmd == "030": # Partition Arm Control
+        elif cmd == "030":  # Partition Arm Control
             success = await self.arm_away()
-        elif cmd == "031": # Partition Arm Control - Stay Arm
+        elif cmd == "031":  # Partition Arm Control - Stay Arm
             success = await self.arm_stay()
-        elif cmd == "032": # Partition Arm Control - Zero Entry Delay
+        elif cmd == "032":  # Partition Arm Control - Zero Entry Delay
             success = await self.arm_max()
-        elif cmd == "040": # Partition Disarm Control
+        elif cmd == "040":  # Partition Disarm Control
             success = await self.disarm()
-        elif cmd == "060": # Trigger Panic Alarm
+        elif cmd == "060":  # Trigger Panic Alarm
             success = await self.trigger_panic_alarm()
-        elif cmd == "071": # Send Keystroke String
+        elif cmd == "071":  # Send Keystroke String
             success = await self.handle_keystroke_sequence()
-        elif cmd == "200": # Send Code
+        elif cmd == "200":  # Send Code
             success = await self.receive_code()
         else:
             log.info(f"Unhandled command ({cmd}); data: {data}")
             return False
 
         return success
 
-
-#    def is_partition_ready(self, partition: int) -> bool:
-#        for zone in self._zone_status:
-#            if not zone:
-#                return False
-#        return True
+    #    def is_partition_ready(self, partition: int) -> bool:
+    #        for zone in self._zone_status:
+    #            if not zone:
+    #                return False
+    #        return True
 
     def get_checksum(self, code, data) -> str:
         checksum = 0
         for ch in code + data:
             checksum = checksum + ord(ch)
-        return "%02X" % (checksum & 0xff)
-
+        return f"{checksum & 255:02X}"
```

Comment:
Why the switch to 255 here?  I personally prefer to use hex when doing bitwise operations.

There's another case where it switched from 0xffff to 0xFFFF so not sure if this was an explicit change by you or a black/flake8 thing?

Reply:
Was a black/flake8 thing.  I'll loop back and address another night.